postman-collection/runner/tests.postman_collection.json

```diff
@@ -0,0 +1,800 @@
+								"type": "text/javascript",
+								"packages": {}
+							}
+						}
+					],
+					"request": {
+						"auth": {
+							"type": "bearer",
+							"bearer": [
+								{
+									"key": "token",
+									"value": "{{access_token}}",
+									"type": "string"
+								}
+							]
+						},
+						"method": "GET",
+						"header": [],
+						"url": {
+							"raw": "{{url}}/api/internal/v2/court_applications/{{application_id}}",
+							"host": [
+								"{{url}}"
+							],
+							"path": [
+								"api",
+								"internal",
+								"v2",
+								"court_applications",
+								"{{application_id}}"
+							]
+						}
+					},
+					"response": []
+				}
+			]
+		},
+		{
+			"name": "Defendants",
+			"item": [
+				{
+					"name": "V1",
+					"item": [
+						{
+							"name": "Defendant",
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"Defendants type is present in the response\", function () {",
+											"    var json_data = pm.response.json();",
+											"    pm.expect(json_data.data.type).to.exist;",
+											"    pm.expect(json_data.data.type).to.eql(\"defendants\");",
+											"});",
+											"",
+											"pm.test(\"Defendant Properties exist\", function () {",
+											"    var json_data = pm.response.json();",
+											"    pm.expect(json_data.data.attributes.name).to.exist;",
+											"    pm.expect(json_data.data.attributes.date_of_birth).to.exist;",
+											"    pm.expect(json_data.data.attributes.national_insurance_number).to.exist;",
+											"    pm.expect(json_data.data.attributes.arrest_summons_number).to.exist;",
+											"});",
+											"",
+											"pm.test(\"Prosecution case in relationships\", function () {",
+											"    var json_data = pm.response.json();",
+											"    pm.expect(json_data.data.relationships).to.exist;",
+											"    pm.expect(json_data.data.relationships.prosecution_case).to.exist;",
+											"    pm.expect(json_data.data.relationships.prosecution_case).to.exist;",
+											"    pm.expect(json_data.data.relationships.prosecution_case).to.be.a('object');",
+											"});"
+										],
+										"type": "text/javascript",
+										"packages": {}
+									}
+								}
+							],
+							"request": {
+								"auth": {
+									"type": "bearer",
+									"bearer": [
+										{
+											"key": "token",
+											"value": "{{access_token}}",
+											"type": "string"
+										}
+									]
+								},
+								"method": "GET",
+								"header": [],
+								"url": {
+									"raw": "{{url}}/api/internal/v1/defendants/{{defendant_summaries_id}}",
+									"host": [
+										"{{url}}"
+									],
+									"path": [
+										"api",
+										"internal",
+										"v1",
+										"defendants",
+										"{{defendant_summaries_id}}"
+									]
+								}
+							},
+							"response": []
+						}
+					]
+				}
+			]
+		}
+	]
+}
```

Comment:
The CDAPI postman tests also interacted with the laa references endpoint - see https://github.com/ministryofjustice/laa-court-data-api/blob/main/postman/collections/laa_references.postman_collection.json. Since that's what we're replacing, at the very least we ought to replicate that. And if we do that, we should also test the court application laa references endpoint. And if we're doing that, maybe we should do the representation order endpoints as well.

Reply:
Yes working on this got some maat id's now thanks to Muthu for providing them to me. but facing some issues as we have updated our error handling which kinda requires a bit more of tact to get the test to work for linking.